_layouts/post.html

```diff
@@ -6,36 +6,41 @@
 {% include feature-image.html %}
 
 <div class="grid-container usa-section">
+  <div class="grid-row grid-gap">
+    <div class="grid-col-2">
```

Comment:
Could you add the `tablet` prefix here so it stacks on mobile

Reply:
👍